menpo/shape/graph.py

```diff
@@ -281,18 +281,6 @@ def _check_vertex(self, vertex):
             raise ValueError('The vertex must be between '
                              '0 and {}.'.format(self.n_vertices - 1))
 
-    def tojson(self):
-        r"""
-        Convert the graph to a dictionary JSON representation.
-
-        Returns
-        -------
-        json : `dict`
-            Dictionary with ``adjacency_array`` key. Suitable or use in the by
-            the `json` standard library package.
-        """
-        return {'adjacency_array': self.adjacency_array.tolist()}
```

Comment:
this is the real problem - it causes the creation of an invalid LJSON file (uses 'adjacency_array' instead of 'connectivity' for the connection information)